packages/mockApi/routes/tug/applications.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line no-unused-vars
+const { errorResp, successResp } = require('../../utils');
+
+module.exports = (req, res) => {
+  const { real_name } = req.body;
+
+  // eslint-disable-next-line no-console
+  console.log(req.body);
+  console.log('real_name', real_name);
+
+  if (real_name === undefined || real_name === '[empty]') {
```

Comment:
Under which scenario the real_name would be '[empty]'?

Reply:
Beside, we may simplify the condition like:

```javascript
if (!real_name || real_name === '[empty]') {
  return errorResp({
      detail: 'something wrong',
      errors: {
        real_name: ['this field is required'],
      },
    })(req, res);
}

return successResp({
  detail: 'success',
})(req, res);
```

Reply:
> Under which scenario the real_name would be '[empty]'?

This magic string is only for testing on mockapi, I will remove this logic before deploying to production environment

Reply:
> This magic string is only for testing on mockapi, I will remove this logic before deploying to production environment

You can use something like `process.env.xxx` to determine the environment.